Makefile

```
@@ -7,11 +7,16 @@ help: Makefile
 	@sed -n 's/^##//p' $< | column -t -s ':' |  sed -e 's/^/ /'
 .PHONY: help
 
-## build: Build celesita-node binary.
+## build: Build celestia-node binary.
 build:
 	@echo "--> Building Celestia"
 	@go build ./cmd/celestia
 
+## install: Builds and installs the celestia-node binary into the GOBIN directory
+install:
+	@echo "--> Installing Celestia"
+	@go install ./cmd/celestia -o $GOBIN
```

I think go install does not need an `-o` flag (in fact I'm not sure this would even work; have you tried locally?). I know this works though:
```suggestion
	@go install ./cmd/celestia
```

---

you are absolutely right